tests/components/unifiprotect/test_camera.py

```diff
@@ -51,7 +51,10 @@ def validate_default_camera_entity(
 
     channel = camera_obj.channels[channel_id]
 
-    entity_name = f"{camera_obj.name} {channel.name}"
+    if channel.name == "Package Camera":
+        entity_name = f"{camera_obj.name} {channel.name}"
+    else:
+        entity_name = f"{camera_obj.name} {channel.name} Resolution Channel"
```

Comment:
Can you get rid of the branching in the test?

Reply:
Not unless I stop testing the name of the camera or go back to a more basic camera entity name (which a lot of people complain about being confusing). The package camera channels are bundled in with the normal ones, so there is no really easy way to identify them other than the name of the channel.

Reply:
I could abstract the core code I use to [generate the base name](https://github.com/home-assistant/core/pull/114088/files#diff-b819c8e3744961cb309ae3feb57a14ebbfc3a62284a1aad26d16ec1109a6c33fR196-R198) and use it in both `camera.py` and the tests, but it would still have a branch regardless.

Reply:
That would be a bit better as it avoids the complexity in the test

Reply:
It does not really change anything since the branching would just be moved from a test helper function to a shared helper function.

The branching is not actually in the test itself.